File: 250316/알파벳에 따른 평가/evaluation-by-alphabet.cpp
#include <iostream>
using namespace std;

int main() {

    char s;
    cin >> s;

    if (s == 'S') {
        cout << "Superior";
    }
    else if (s == 'A') {
        cout << "Excellent";
    }
    else if (s == 'B') {
        cout << "Good";
    }
    else if (s == 'C') {
        cout << "Usually";
    }
    else if (s == 'D') {
        cout << "Effort";
    }
    else {
        cout << "Failure";
    }

    return 0;
}

File: 250316/알파벳에 따른 평가/evaluation-by-alphabet.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        String s = sc.next();

        if(s.charAt(0) == 'S') {
            System.out.println("Superior");
        }
        else if (s.charAt(0) == 'A') {
            System.out.println("Excellent");
        }
        else if (s.charAt(0) == 'B') {
            System.out.println("Good");
        }
        else if (s.charAt(0) == 'C') {
            System.out.println("Usually");
        }
        else if (s.charAt(0) == 'D') {
            System.out.println("Effort");
        }
        else {
            System.out.println("Failure");
        }
    }
}